libs/ui-lib/lib/ocm/components/AddHosts/day2Wizard/Day2ClusterDetails.tsx

```diff
@@ -242,7 +242,10 @@ const Day2ClusterDetails = () => {
                 )}
                 <GridItem>
                   <Day2HostStaticIpConfigurations
-                    isDisabled={day2Cluster.platform?.type === 'oci'}
+                    isDisabled={
+                      day2Cluster.platform?.type === 'external' &&
```

Comment:
We could extract the logic that given a `cluster`, indicates if it's of a given platform.

I see a similar construct being used in several places (twice in this file, also later in `NetworkConfiguration`, `ClusterService` etc), and it would avoid possible typos etc.